manifests/k/KuroGames/WutheringWaves/1.5.1.0/KuroGames.WutheringWaves.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Created with komac v2.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: KuroGames.WutheringWaves
+PackageVersion: 1.5.1.0
+PackageLocale: en-US
+Publisher: 广州库洛科技有限公司

```

```suggestion
Publisher: KURO TECHNOLOGY (HONG KONG) CO., LTD.
```
This should match Control Panel